### PR TITLE
OPS: Idle timer

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/tyfuu03wtfhjde7wcymwb0g08m

[download apk](http://filestorage.bluewallet.io:1488/idle-1611939437.apk)

File: package.json [hunk 1]
```diff
@@ -132,6 +131,7 @@
     "react-native-gesture-handler": "1.8.0",
     "react-native-handoff": "git+https://github.com/marcosrdz/react-native-handoff.git#f5becc63f3e36bf2da1ed1fc60fc690323e73602",
     "react-native-haptic-feedback": "1.11.0",
+    "react-native-idle-timer": "git+https://github.com/BlueWallet/react-native-idle-timer.git#8587876d68ab5920e79619726aeca9e672beaf2b",
```
Comment (Overtorment) [inline package.json:134]: dont we need to somehow call this..?

**Review by Overtorment — CHANGES_REQUESTED** [review]

**Comment by Overtorment** [comment]
it makes sense disabling sleep ONLY during import procedure. 
phone should lock if just idle for security reasons

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/pge1wqb4k5v3kjp82r339yuxyw

[download apk](http://filestorage.bluewallet.io:1488/idle-1611953881.apk)

**Review by Overtorment — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**